### PR TITLE
[Reviewer: Andy] Add new event for quorum form failures

### PR DESCRIPTION
**Pull request by Tom-Rees**
Hi Andy,

This is the sas-bundle fix associated with [this PR](https://github.com/Metaswitch/cpp-common/pull/349). Would you mind reviewing it please?

Thanks,
Tom

### CODE REVIEW

File: clearwater_sas_resource_bundle.yaml [hunk 1]
```diff
@@ -1682,6 +1688,10 @@ enums:
     0: 'Temporary error response threshold exceeded'
     1: 'Permanent error response received'
 
+  QUORUM_CONSISTENCY_LEVELS:
+    0: 'LOCAL_QUORUM'
+    1: 'QUORUM'
```
Comment (bossmc) [inline clearwater_sas_resource_bundle.yaml]: Presumably these might need updating if we decide to switch to using `::cass::ConsistencyLevel::???` values?

File: clearwater_sas_resource_bundle.yaml [hunk 2]
```diff
@@ -371,6 +371,12 @@ events:
       Error text: {{ var_data[0] }}
     level:   40
 
+  0x000400:
+    summary: "Failed to form quorum"
```
Comment (bossmc) [inline clearwater_sas_resource_bundle.yaml]: I much prefer the terminology in the details to the summary, probably for the summary, I'd say something like "Cassandra operation failed due to lack of quorum" then leave the details as is.

**Comment by bossmc** [comment]
Looks mostly fine.

**Comment by Tom-Rees** [comment]
Changes made. Slightly re-worded sas log summary to make cause of error more clear.
Enum values taken from [here](http://grepcode.com/file/repo1.maven.org/maven2/org.apache.cassandra/cassandra-thrift/2.0.5/org/apache/cassandra/thrift/ConsistencyLevel.java) and these look consistent with the values I saw in testing.

**Comment by bossmc** [comment]
The canonical location for the numbers is https://github.com/apache/cassandra/blob/trunk/interface/cassandra.thrift#L231-242 which agrees with what you put.

**Pull request merged**